database/migrations/2017_12_04_113055_create_children_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateChildrenTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('children', function (Blueprint $table) {
+            $table->increments('id');
+            $table->dateTime('dob');
+            $table->boolean('born')->default(false);
```

Comment:
Could this be appended? If there is a bith date in the past then yes! If not, no! ?

Reply:
It's a flag to say "this person has been acknowledged as actually being 'born'". 
While the future DoB was considered. They wanted a flag to say "we've been told this person has been born". There may be a UI/Print notice to check with the family that.